universal-login-commons/lib/utils/types.ts

```diff
@@ -32,4 +32,16 @@ export interface DeviceInfo {
   os: string;
   browser: string;
   time: string;
+}
+
+export interface ITransactionQueueStore {
```

Comment:
1) Move to seperate file
2) 
```ts
class TransactionEntity {
    id: string;
    hash: string | undefined;
    error: string | undefined;
    message: Partial<utils.Transaction>;
  }
```
3.  `getNext: () => Promise<TransactionEntity?>`
4. Create `services/transactions`